### DISCUSSION TITLE
REMOTE can not connect  using -F /dev/nul

### DISCUSSION THREAD

**Discussion by idhamari** [microsoft/vscode-discussions#537] [Extension Development QnA]
Dear vscode community,

I am trying to connect to a remote machine via Remote extension. I can connect using the terminal  e.g. 

   ssh -v -F /dev/null user@example.come

But I could not connect using the Remote extension. 

             debug1: Server host key:  examplekey
             Load key "/dev/null": error in libcrypto
             user@example.com: Permission denied (publickey,password).
             local-server-1> ssh child died, shutting down

I tried to modify the config file based on googling but nothing works e.g. 

             IdentityFile /dev/null
  
or 

           StrictHostKeyChecking no
  UserKnownHostsFile /dev/null

or 
  UserKnownHostsFile ~/.ssh/known_hosts

  PubkeyAuthentication no
  PreferredAuthentications password

or 
 
    StrictHostKeyChecking no
    UserKnownHostsFile=/dev/null 

Any suggestions?

I am using visual code installed in Ubuntu machine and trying to connect to another Ubuntu machone

**Comment by isidorn** [comment]
I suggest to open a new issue in the https://github.com/microsoft/vscode-remote-release
